frontend/src/common/styles/container.css

```diff
@@ -0,0 +1,24 @@
+.main-container {
+    display: flex;
+    min-height: 70vh;
+}
+
+.content-container {
+    margin: auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.components-container {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+}
+
+.card-payment {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: #F3F3F3;
```

Comment:
Don't use this ugly grey background from win 98. Look at the design. Background is white but borders have shadows from all sides